### PR TITLE
Plant details screen

### LINKED ISSUE #33: Implement Plant details screen
Implement plant detail screen:
- [x] Open detail screen when user taps on plant in household view.
- [x] Display properties from Plant model (ordered):
   1. [x] Nickname
   2. [x] Health status
   3. [x] Image
   4. [x] Common name
   5. [x] Scientific name
   6. [x] Description
   7. [x] Care-taking instruction
   8. [x] Desired temperature
   9. [x] Watering Frequency
   10. [x] Care intensity
- [x] Navigating back directs user to the household view.

### PR DESCRIPTION
**Pull request by Inn0**
## 🧙🏻‍♂️ Describe your changes

- Implemented plant details screen

## 🔗 Issue that is implemented here 

- Closes #33 

## 📸 Screenshot or screen recording of working solution

|   #   | File  |
| ----- | ----- |
|   1   |  IMG_0019 |
|  2 | IMG_0020 |

## 👮🏻‍♂️ Check yourself

- [x] I have performed a self-review of my code
- [x] I have resolved all TODOs in this branch
- [x] Changes are covered by unit tests

### CODE REVIEW

File: BloomBuddyMobile/View/General/ImageView.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            @unknown default:
+                EmptyView()
+            }
+        }
+    }
+}
+
+#Preview {
+    ImageView(url: "https://i1.sndcdn.com/artworks-x8zI2HVC2pnkK7F5-4xKLyA-t500x500.jpg")
+}
```
Comment (ivansdivans) [inline BloomBuddyMobile/View/General/ImageView.swift:35]: LMAO

File: BloomBuddyMobile/View/Plant/PlantDetailView.swift [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                Text(plant.description)
+                Spacer()
+            }
+            Spacer(minLength: 20)
+            HStack {
+                Text("Caretaking")
+                    .font(.title)
+                    .fontWeight(.bold)
+                Spacer()
+            }.padding(.vertical, 2)
```
Comment (ivansdivans) [inline BloomBuddyMobile/View/Plant/PlantDetailView.swift:43]: I would prefer modifiers to be on the new line. This way we can keep code base consistent :)

Comment (Inn0) [reply BloomBuddyMobile/View/Plant/PlantDetailView.swift]: Agreed, nice catch. 'T was a long day. I changed it.

**Review by ivansdivans — APPROVED** [review]
Good to go 🚀

**Pull request merged**